Sources/SwiftDocC/SwiftDocC.docc/SwiftDocC/LinkResolution.md

```diff
@@ -22,13 +22,25 @@ doc://com.example/path/to/documentation/page#optional-heading
       bundle ID     path in docs hierarchy    heading name 
 ```
 
-## Resolving a documentation link
+Both types of links can be used in a relative or absolute way. Absolute symbol links have a leading slash (`/`) and must start with the module they are referring to, for example ` ``/MyModule/MyClass/myProperty`` `.
```

Comment:
This first sentence seems either incorrect to me or at least confusing to me. The symbol link part is correct but leading slashes doesn't have—or at least I feel that they shouldn't have—any effect on "doc" scheme links. 

I described some of the "doc" scheme behaviors in [this forum comment][1]. 

Also, I'm not super happy about the term "absolute" for links because it already has specific meanings in other context. This could be avoided by instead focusing on the behavior and saying:

> Symbol links that start with a leading slash resolve relative to the scope of modules instead of the scope of the page that contains the link.



[1]: https://forums.swift.org/t/combined-documentation-of-multiple-targets/59579/18

Reply:
So "doc" scheme links are always relative, but the "host name" component (if present) stops them from being matched to a path that begins deeper down the tree?